### ISSUE TITLE
(a11y) Content structure

### ISSUE THREAD

**Issue by bbsmooth** [DBenMoshe/contrast-ratio-repo#63]
Something that should be addressed early in the process is content structure. The `h1` is good, and the first `h2` below it is fine, but then the next heading is an `h3`? I'm wondering if "How much contrast do I need?" is really a subheading of "How does it work?" These seem like completely separate topics. One tells you how the app works and the other describes the WCAG rules for contrast. Personally, I think the "How much contrast do I need?" should be at the same heading level (`h2`) as "How does it work?". And if this change is made then of course the subheadings under "How much contrast do I need?" will need to be moved up one as well.

I also think there might be a case for adding headings for the results. Right under the inputs you have a row of results. These feel like they should be grouped together under a `Results` heading (`h2`). And then each type of result would be an `h3` heading. This will make it easier for screen reader users to jump to the result they want (screen reader users can navigate a page by headings). I could also see a case being made for using a `dl` element instead of `h3` headings. But I would still keep the `Results` heading. 

Speaking of the `Results` heading, you can visually hide it so that it will be available to screen reader users without being shown on the page. I personally don't think there is anything wrong with having the `Results` heading on the page, but I understand that you might not want it there.

This is just to get the conversation going. I am not demanding anything here. Accessibility can be met in a variety of ways. My approach is to point out a potential issue, explain why it's an issue, give you some options/suggestions, and answer any questions. I will try to only put my foot down when there is a really big accessibility violation that needs to be fixed.

**Comment by jdwilkin4** [comment]
Thanks for the feedback @bbsmooth ! 
I think all of those changes for the headings sound reasonable to me.

**Comment by AndreyDmitriev39r** [comment]
>One tells you how the app works and the other describes the WCAG rules for contrast. Personally, I think the "How much contrast do I need?" should be at the same heading level (h2) as "How does it work?". And if this change is made then of course the subheadings under "How much contrast do I need?" will need to be moved up one as well.

**Comment by jdwilkin4** [comment]
Created a new issue to address what was brought up in this discussion.
#69
